utils/debug/runtime.go

```diff
@@ -0,0 +1,16 @@
+package debug
+
+import (
+	"runtime/metrics"
+)
+
+// GetHeapAllocsBytes returns the value of /gc/heap/allocs:bytes.
+func GetHeapAllocsBytes() uint64 {
+	sample := []metrics.Sample{{Name: "/gc/heap/allocs:bytes"}}
```

Comment:
maybe just alloc this once globally

Reply:
since it is modified in place i think it is better to leave it like this for now since the locking cost will be higher. Also, hopefully the compiler is smart enough to not escape this one to the heap.